### DISCUSSION TITLE
Add support for `fftshift` function from NumPy

### DISCUSSION THREAD

**Discussion by belyan** [josdejong/mathjs#3191] [Ideas]
I need this function in my project, but it is missed in `math,js` now, so I forced to import [fftshift](https://www.npmjs.com/package/fftshift) library to use it.

Here is documentation of this function from NumPy site:
https://numpy.org/doc/stable/reference/generated/numpy.fft.fftshift.html

Here is source code on Python:
https://github.com/numpy/numpy/blob/v1.26.0/numpy/fft/helper.py#L19-L73

Here is an example of implementation on JavaScript:
https://unpkg.com/browse/fftshift@1.0.1/build/index.js

**Comment by josdejong** [comment]
Thaks for your suggestion Yuri! 

Anyone interested in implementing a function `fftshift`? Help would be welcome.
